src/vllm_tgis_adapter/grpc/grpc_server.py

```diff
@@ -461,7 +461,12 @@ def _convert_input_details(
 
         if resp_options.input_text:
             response.text = (
-                result.prompt if not response.text else result.prompt + response.text
+                result.prompt
+                if not response.text
```

Comment:
So if  `response.text` is None, we have the prompt text in `result.prompt`, but if it is not None, the `result.prompt` is discarded or never set?

Reply:
Hmm, good point. How can I test out the `response.text` as None?